docs/en/dora/Deploy-Alluxio-Cluster.md

```diff
@@ -62,6 +62,21 @@ The cache store used by Dora cache nodes is currently hardcoded to be the paged
 [documentation](https://github.com/Alluxio/alluxio/blob/dora/docs/en/core-services/Caching.md#experimental-paging-worker-storage)
 on how to configure the paged block store.
 
+## Dora on Kubernetes
+
+The existing Alluxio helm chart works with Dora cluster except Alluxio Fuse. Note that:
+1. Set your image to `alluxio/alluxio:2.9.1-gamma`.
```

Comment:
version is 291-gamma instead of 2.9.1-gamma

Reply:
For image name it is possible to use `2.9.1-gamma`. Do you prefer to using `291-gamma`?

Reply:
Synced offline the version should be 291-gamma instead of 2.9.1-gamma.